testing/AirInterfacePowerSaver+data.json

```diff
@@ -2113,12 +2119,12 @@
             ]
           },
           {
-            "uuid": "aips-0-0-5-op-c-is-mwdg-1-0-0-500",
+            "uuid": "aips-0-0-10-op-c-is-mwdg-1-0-0-500",
             "ltp-direction": "core-model-1-4:TERMINATION_DIRECTION_SINK",
             "client-ltp": [
             ],
             "server-ltp": [
-              "aips-0-0-5-http-c-mwdi-1-0-1-000"
+              "aips-0-0-10-http-c-mwdg-1-1-1-000"
```

Comment:
the latest version of mwdg is 1.01. So, the release-number component in uuid of mwdg and http-sever/release-number could be updated accordingly